docs/vgclog_examples.md

```diff
@@ -0,0 +1,594 @@
+
+<mem-info id="6" free="802568640" total="1073741824" percent="74">
+
+<mem type="nursery" free="0" total="268435456" percent="0">
+
+<mem type="allocate" free="0" total="134217728" percent="0" />
+
+<mem type="survivor" free="0" total="134217728" percent="0" />
+
+</mem>
+
+<mem type="tenure" free="802568640" total="805306368" percent="99">
+
+<mem type="soa" free="762302912" total="765040640" percent="99" />
+
+<mem type="loa" free="40265728" total="40265728" percent="100" />
+
+</mem>
+
+<remembered-set count="30758" />
+
+</mem-info>
+
+</gc-start>
+
+<allocation-stats totalBytes="136862352" >
+
+<allocated-bytes non-tlh="7097008" tlh="129765344" />
+
+<largest-consumer threadName="Start Level: Equinox Container: c25c3c67-836a-49e7-9949-a16d2c2a5a4d" threadId="000000000045E300" bytes="46137952" />
+
+</allocation-stats>
+```
+
+In this example, the *nursery* area has no free space available in either the *allocate* or *survivor* spaces. The *tenure* area has 99% free in the small object area (SOA). The large object area (LOA) contains no objects.
+
+The *scavenge* GC operation starts with the `<gc-op>` element; child elements record details about the operation. For example, `<scavenger-info>` shows that the *tenure age* is set to `1`. For a *tenure age* of `1`, any objects that are copied between the *allocate* and *survivor* space once are moved to the *tenure* area during the next *scavenge* operation.
+
+```xml
+<gc-op id="7" type="scavenge" timems="30.675" contextid="4" timestamp="2020-10-18T13:27:09.634">
+
+<scavenger-info tenureage="1" tenuremask="fffe" tiltratio="50" />
+
+<memory-copied type="nursery" objects="461787" bytes="25522536" bytesdiscarded="367240" />
+
+<finalization candidates="317" enqueued="157" />
+
+<ownableSynchronizers candidates="6711" cleared="2579" />
+
+<references type="soft" candidates="9233" cleared="0" enqueued="0" dynamicThreshold="32" maxThreshold="32" />
+
+<references type="weak" candidates="13149" cleared="4781" enqueued="4495" />
+<references type="phantom" candidates="359" cleared="8" enqueued="8" />
+
+<object-monitors candidates="79" cleared="36" />
+
+</gc-op>
+```
+
+The `<memory-copied>` element indicates that 25 MB (25522536) of reachable objects were moved by the *scavenge* operation. For more information about how the scavenge operation acts on the heap, see [`gencon` policy(default)](gc.md#gencon-policy-default).
+
+The end of the increment is recorded with `<gc-end>` and provides another snapshot of memory allocation on the heap, similar to `<gc-start>`. In the example, 40% of the *nursery* area is now free as a result of the *scavenge* operation. All reachable objects are in the *survivor* area, freeing up space in the *allocate* area for new objects. The *tenure* area remains at 99% free, which means that the *scavenge* operation reclaimed memory from the *nursery* area; no objects reached the specified *tenure age* that triggers a move to the *tenure* area.
```

Comment:
As mentioned above, it's more common to see some object movement to tenure, in which case tenure free will slightly drop (not necessarily in percentages due to rounding, but the bytes free should drop). I recommend you update the example, and reflect the changes in text too.